sdk/lib/opentelemetry/sdk/trace/export/batch_span_processor.rb

```diff
@@ -82,7 +87,7 @@ def shutdown
 
           private
 
-          attr_reader :spans, :max_queue_size, :batch_size
+          attr_reader :spans, :max_queue_size, :batch_size, :export_timeout_seconds
```

Comment:
I don't think we need to expose this (other than for testing, and I'd rather do that another way).

Reply:
I've removed the reader, as well as rewritten the test. :heavy_check_mark: